src/routes/menus/SideMenuNavLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavLink } from "react-router-dom"
+import { Text } from "../../components/typography/Text"
+import { ThemeColor } from "../../theme"
+
+interface ISideMenuNavLinkProps {
```

Comment:
NavLink že ima definirane propertyje. Namesto da en po en property definiraš sam, poglej kakšne propse ima NavLink, kakšen interface definira in extendaj tvoje propse z nav linkovim.

torej to mora zgledat nekak tak

`ISideMenuNavLinkProps extends PropsiOdNavLinka {}`